scripts/list_needed_libs.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+set -e
+
+# this file lists all dynamic dependenices of executables in gprofiler/resources.
+# we use it to let staticx know which libraries it should pack inside.
+# (staticx knows to pack the libraries used by the executable we're packing. it doesn't know
+# which executables are to be used by it)
+
+BINS=$(find gprofiler/resources -executable -type f)
+
+libs=
+
+for f in $BINS ; do
+    set +e
+    ldd_output=$(ldd $f 2>&1)
+    ret=$?
+    set -e
+
+    if [ $ret -eq 0 ]; then
+        if [[ "$ldd_output" == *" not found"* ]]; then
+            >&2 echo "missing libs/symbols for binary $f"
+            >&2 cat <<< "$ldd_output"
```

Comment:
```suggestion
            >&2 echo "$ldd_output"
```

Reply:
I used `cat` because `echo` got rid of line ends for me. Tested now again and it looks alright. Weird

Reply:
Will switch to it

Reply:
Because it was split with `$IFS`

Reply:
I tried on my shell now:
```
$ cat a
a
b
c
$ x=$(cat a)
$ echo "$x"
a
b
c
$ cat <<< "$x"
a
b
c
```

without touching `IFS`

Reply:
Ah right, it's because of the quoting.